joomla/joomla-cms
# [5.1] Convert mod_feed to service provider

### Summary of Changes
 
Converts the feed site module to service provider

### Testing Instructions

- create site module `feed`
- check site output

Note: system test added

### Actual result BEFORE applying this Pull Request

module works
![image](https://github.com/joomla/joomla-cms/assets/66922325/bb4c66fe-b8f9-43a7-aacb-423458c4d58f)

### Expected result AFTER applying this Pull Request

module works
![image](https://github.com/joomla/joomla-cms/assets/66922325/bb4c66fe-b8f9-43a7-aacb-423458c4d58f)

### Link to documentations
Please select:
